pkg/asset/installconfig/powervs/validation.go

```diff
@@ -253,3 +253,26 @@ func ValidateResourceGroup(client API, ic *types.InstallConfig) error {
 
 	return nil
 }
+
+// ValidateSystemTypeForRegion checks if the specified sysType is available in the target region.
+func ValidateSystemTypeForRegion(client API, ic *types.InstallConfig) error {
+	if ic.ControlPlane.Platform.PowerVS == nil || ic.ControlPlane.Platform.PowerVS.SysType == "" {
```

Comment:
Probably need the `ControlPlane == nil` check here too.

Reply:
Addressed in latest force-push. PTAL.